patternfly/patternfly-elements
# feat: [pfe-styles] ordered list styles

## <Pull request description>

This update adds a new Sass mixin called `ordered-list-primary`.  It is leveraged by pfe-styles to create a new component called `ol.pfe-list--primary`.

![Screen Shot 2021-04-29 at 5 58 06 PM](https://user-images.githubusercontent.com/3428964/116623602-7908ec00-a914-11eb-99ce-33bac3ec4245.png)



### Related issues

- (#1565) Include a primary ordered list style for pfe-styles


### Preview


Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-1571--patternfly-elements.netlify.app/elements/pfe-styles/demo/) (Scroll down to Primary Ordered List header)


### Testing instructions




- *Branch test*
  1. Go to the [demo page](https://deploy-preview-1571--patternfly-elements.netlify.app/elements/pfe-styles/demo/)
  2. Scroll down to "Primary Ordered List"
  - [x] **It should match mockup shown above.**
  - [x] Open you browser and past in the code below. **verify that the list items look like [this screenshot](https://user-images.githubusercontent.com/3428964/118015048-db54ea00-b321-11eb-84bd-eff7fe80edb6.png)**
```
const list = document.querySelector("ol.pfe-list--primary");
list.style.setProperty("--pfe-list-ol--primary--gap", "0px");
list.style.setProperty("--pfe-list-ol--primary--spacer", "0px");
list.style.setProperty("--pfe-list-ol--primary__circle--size", "56px");
list.style.setProperty("--pfe-list-ol--primary__circle--color", "0,128,0");
list.style.setProperty("--pfe-list-ol--primary__circle--BackgroundColor", "yellow");
list.style.setProperty("--pfe-list-ol--primary__circle--FontSize", "33px");
```



#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
